lib/octokit/client/organizations.rb

```diff
@@ -595,6 +595,89 @@ def remove_organization_membership(org, options = {})
         user = options.delete(:user)
         user && boolean_from_response(:delete, "orgs/#{org}/memberships/#{user}", options)
       end
+
+      # Initiates the generation of a migration archive.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @option options [Array<String>] :repositories Repositories for the organization.
+      # @option options [Boolean, optional] :lock_repositories Indicates whether repositories should be locked during migration
+      # @return [Sawyer::Resource] Hash representing the new migration.
+      # @see https://developer.github.com/v3/orgs/teams/#create-team
+      # @example
+      #   @client.start_migration('github', {
+      #     :repositories => ['github/dotfiles']
+      #   })
+      # @see https://developer.github.com/v3/orgs/migrations/#start-a-migration
+      def start_migration(org, options = {})
+        options = ensure_api_media_type(:migrations, options)
+        post "orgs/#{org}/migrations", options
+      end
+
+      # Lists the most recent migrations.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @return [Array<Sawyer::Resource>] Array of migration resources.
+      # @see https://developer.github.com/v3/orgs/migrations/#get-a-list-of-migrations
+      def migrations(org, options = {})
+        options = ensure_api_media_type(:migrations, options)
+        paginate "orgs/#{org}/migrations", options
+      end
+
+      # Fetches the status of a migration.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @param id [Integer] ID number of the migration.
+      # @see https://developer.github.com/v3/orgs/migrations/#get-a-list-of-migrations
+      def migration_status(org, id, options = {})
+        options = ensure_api_media_type(:migrations, options)
+        get "orgs/#{org}/migrations/#{id}", options
+      end
+
+      # Fetches the URL to a migration archive.
+      #
+      # Requires authenticated organization owner.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @param id [Integer] ID number of the migration.
+      # @see https://developer.github.com/v3/orgs/migrations/#download-a-migration-archive
+      def download_migration_archive(org, id, options = {})
+        options = ensure_api_media_type(:migrations, options)
+        url = "orgs/#{org}/migrations/#{id}/archive"
+
+        response = client_without_redirects(options).get(url)
+        response.body
```

Comment:
I just checked, and you're right, `Location` provides the same information as the body:

```
   body=
    "https://s3.amazonaws.com/github-cloud/migration/102/89?response-content-disposition=filename%3Db8902e90-3c6b-11e5-8f00-4be3b3f48d46.tar.gz&response-content-type=application/x-gzip&AWSAccessKeyId=...",
...
   response_headers=
    {"server"=>"GitHub.com",
     "date"=>"Thu, 06 Aug 2015 18:50:12 GMT",
...
     "location"=>
      "https://s3.amazonaws.com/github-cloud/migration/102/89?response-content-disposition=filename%3Db8902e90-3c6b-11e5-8f00-4be3b3f48d46.tar.gz&response-content-type=application/x-gzip&AWSAccessKeyId=...",
     "x-xss-protection"=>"1; mode=block",
```

Do you have a preference in terms of longer term stability?